src/libponyc/codegen/gentype.c

```diff
@@ -156,7 +156,7 @@ static bool make_opaque_struct(compile_t* c, reach_type_t* t)
           c_t->mem_type = c->void_ptr;
           return true;
         }
-        else if(name == c->str_Maybe)
+        else if (name == c->str_NullablePointer)
```

Comment:
whitespace after if should be removed.

Reply:
i dont believe this change should have been made. Maybe isn't always in reference to Nullable pointer.

It's used in the C codebase for things that might be null in the compiler which is outside the scope of the RFC.